microsoft-edge/devtools-guide-chromium/memory-problems/index.md

```diff
@@ -232,14 +222,19 @@ To record an Allocation instrumentation on timeline:
 ![Object details](./index-images/allocation-timeline-snapshot-focused-constructor-expanded.png)
 
 
+See also:
+* [Use Allocation instrumentation on timeline ("Allocations on timeline" profiling type)](./allocation-profiler.md)
+
+
+<!--[3]-->
 <!-- ====================================================================== -->
-## Investigate memory allocation by function
+## Investigate memory allocation by function ("Allocation sampling" profiling type)
```

Comment:
move content into article:
"Speed up JavaScript runtime ("Allocation sampling" profiling type)"
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/rendering-tools/js-runtime?branch=pr-en-us-3282

present article is too long for LMC, and dup of section + article is confusing for maintainers & users

Reply:
status re: this section
Retained h2 heading for now, only containing a link to the dedicated article.
In table at top of this Mem tool article, linked to dedicated article, not to this empty h2 section.

Table at top of Mem tool article is too sparse though, needs to provide concise but effective explanation of when to use which tool or which prof type of Mem tool.  eg below) is the old text from Mem tool's home page, to give such contrast.  But need you to write fresh advice/ framing/ positioning, on when to use which tool/prof type, for which kind of task/analysis.

presently hidden comment at top of Mem tool article:

* **Heap snapshot** - Heap snapshot profiles show memory distribution among your page's JavaScript objects and related DOM nodes.

* **Allocation instrumentation on timeline** - Allocation timelines show instrumented JavaScript memory allocations over time.  Once profile is recorded you can select a time interval to see objects that were allocated within it and still alive by the end of recording.  Use this profile type to isolate memory leaks.

* **Allocation sampling** - Record memory allocations using sample method.  This profile type has minimal performance overhead and can be used for long running operations.  It provides good approximation of allocations broken down by JavaScript execution stack.

* **Detached elements** - _no such option button & text existed; assume the above intentions have been modified by adding this prof type and are now outdated advice/positioning_

@captainbrosset

Reply:
here are the overview explanations in latest Mem tool home page, how should Mem tool article summarize/contrast these prof types, along w/ Perf tool > Mem checkbox, and Edge Task Manager?

![image](https://github.com/user-attachments/assets/ae2dc668-cea9-42fa-ad63-fb0049affb0a)

@captainbrosset